authorizer.js

```diff
@@ -0,0 +1,3 @@
+const authoriser = require('node-lambda-authorizer')({ jwtSecret: process.env.jwtsecret, allowedGroups: process.env.allowedGroups.split(",") });
+
+exports.handler = authoriser;
```

Comment:
love it! could even move to one line - exports.handler = require('node-lambda-etc...